src/ci/docker/host-x86_64/dist-loongarch64-linux/Dockerfile

```diff
@@ -25,5 +25,11 @@ ENV CC_loongarch64_unknown_linux_gnu=loongarch64-unknown-linux-gnu-gcc \
 
 ENV HOSTS=loongarch64-unknown-linux-gnu
 
-ENV RUST_CONFIGURE_ARGS --enable-extended --disable-docs
+ENV RUST_CONFIGURE_ARGS \
+      --enable-extended \
+      --enable-full-tools \
+      --enable-profiler \
+      --enable-sanitizers \
```

Comment:
We don't have sanitizers properly enumerated in our target definition, thus this line does not make sense to me.

https://rustc-dev-guide.rust-lang.org/sanitizers.html?highlight=new%20target#enabling-sanitizer-on-a-new-target

Reply:
Ah, good point, didn't catch that. https://github.com/rust-lang/rust/blob/1.79.0/compiler/rustc_target/src/spec/targets/loongarch64_unknown_linux_gnu.rs vs e.g. https://github.com/rust-lang/rust/blob/1.79.0/compiler/rustc_target/src/spec/targets/x86_64_unknown_linux_gnu.rs#L11.

Reply:
Ah, good catch! I'll save the sanitizers enablement for another PR. I'll remove them when I get to use my laptop in 1 or 2 hours.